### ISSUE TITLE
AWS CloudWatch in console not working due to logs.eu-west-1.amazonaws.com being blocked

### ISSUE THREAD

**Issue by bbartlomiej** [AdguardTeam/AdGuardSDNSFilter#953]
### Steps to reproduce
1. Enable default https://adguardteam.github.io/AdGuardSDNSFilter/Filters/filter.txt filter
2. Go to AWS Console
3. Go to CloudWatch
4. Try to list logs in log groups

Please remove legitimate AWS service from blocklist. There's an entry in above list for logs.eu-west-1.amazonaws.com - this should be removed.

### Expected behavior
View logs in log group of CloudWatch

### Actual behavior
Console can't enumerate logs on web page. Reason being logs.eu-west-1.amazonaws.com is blocked on list.

Screenshot:
![1](https://user-images.githubusercontent.com/4271832/168834663-239e9b81-9277-40d3-8233-565bab0464f9.PNG)

![2](https://user-images.githubusercontent.com/4271832/168834672-14ea6590-a054-4027-a2c7-8926705f83b1.PNG)

![3](https://user-images.githubusercontent.com/4271832/168835352-a5087490-9d4e-4dde-9848-8184120412c0.PNG)

### Your environment

|   Description  |     Value    |
| -------------- | ------------ |
| DNS server in use: | AdGuardHome v0.107.6
| How did you setup DNS configuration: | router
| Device model: mikrotik: | hEX gr3
| Operating system and version: | any

**Comment by Alex-302** [comment]
Duplicate of https://github.com/AdguardTeam/AdguardFilters/issues/117149
